README.md

```diff
@@ -4,53 +4,46 @@ A [Roc](https://github.com/roc-lang/roc) library for random number generation
 
 ## Status
 
-This works, but there's much more it could do.
+Forked from [JanCVanB/roc-random](https://github.com/JanCVanB/roc-random), updated to build package and documentation, and do some maintenance.
```

Comment:
No longer a fork, since you're copying back to the original repo.

Reply:
I've now fixed this via https://github.com/JanCVanB/roc-random/pull/26